src/_sass/_main_nav.scss

```diff
@@ -0,0 +1,532 @@
+$nav-blue: #4052a4;
+$nav-bottom-shadow: #f5f5f5;
+$mobile-nav-border: #e5e5e5;
+$hover-color: #f9f9f9;
```

Comment:
these can be declared in main.scss and the names generalized. i'm using these in career-lab